exporter/zipkinexporter/zipkin_test.go

```diff
@@ -339,3 +343,60 @@ const zipkinSpansJSONJavaLibrary = `
   "duration": 207000
 }]
 `
+
+// The rest of the fields should match up exactly
+func TestZipkinExporter_roundtripProto(t *testing.T) {
+	buf := new(bytes.Buffer)
+	var contentType string
+	cst := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		io.Copy(buf, r.Body)
+		contentType = r.Header.Get("Content-Type")
+		r.Body.Close()
+	}))
+	defer cst.Close()
+
+	tes, err := newZipkinExporter(cst.URL, "", time.Millisecond, true)
+	if err != nil {
```

Comment:
nit: we are trying to make the tests less verbose by using require/assert as appropriate (like you did in the end of the test). Could you please replace these `if` statement checks with require/assert as appropriate? I know you probably got this from other test still using most `if` statements...